Add post about joining the Open Source Collective (#6)
pyodide/pyodide-blog: content/posts/pyodide-joins-open-source-collective.md

---
title: "Pyodide becomes member of the Open Source Collective"
date: 2021-11-10T13:13:04+01:00
draft: true
tags: ["announcement"]

> author: "Roman Yurchak on behalf of the Pyodide team"

Author: We need to decide how we formulate this in project announcements blog posts. We could just put "The Pyodide team" as the author, but I feel the person who writes the post should still appear somewhere. Also in practice most of these would likely be co-written / changed through review, so here it would likely be Hood and me?

showToc: false
TocOpen: false
draft: true
hidemeta: false
comments: false
# description: "Desc Text."
# canonicalURL: "https://canonical.url/to/page"
disableHLJS: true # to disable highlightjs
disableShare: false
hideSummary: false
searchHidden: true
ShowReadingTime: true
ShowBreadCrumbs: true
ShowPostNavLinks: true
#cover:
#    image: "<image path/url>" # image path/url
#    alt: "<alt text>" # alt text
#    caption: "<text>" # display caption under cover
#    relative: false # when using page bundles set this to true
#    hidden: true # only hide on current single page
---

We would like to announce that Pyodide is now member of the [Open Source
Collective (OSC)](https://www.oscollective.org/), which will act as project's
fiscal host.

> The Pyodide project, that compiles Python and the scientific stack to
> WebAssembly, was created in 2018 by Michael

Reviewer: Flows better as two separate sentences IMO.
```suggestion
The Pyodide project compiles Python and the scientific stack to WebAssembly using Emscripten. 
Pyodide was created in 2018 by Michael
```

Droettboom at Mozilla as part of the [Iodide

> project](https://github.com/iodide-project/iodide). Later in 2021 [Pyodide
> become an indepenent open-source
> project](https://hacks.mozilla.org/2021/04/pyodide-spin-out-and-0-17-release/),
> and the current annoucement completes this transition. By selecting OSC as our
> fiscal host, Pyodide will be able to recieve donnations and accepts
> sponsorships which would help long term project sustainability.

Reviewer: Fix typos and improve flow of last sentence
```suggestion
project](https://github.com/iodide-project/iodide). Later in 2021 [Pyodide
become an independent open-source
project](https://hacks.mozilla.org/2021/04/pyodide-spin-out-and-0-17-release/),
and the current announcement completes this transition. Pyodide will now be able to receive donations and accept
sponsorships though OSC, which will help long term project sustainability.
```

We are now accepting donations at:
 - the OSC project page: [opencollective.com/pyodide](https://opencollective.com/pyodide)

>  - or via Github Sponsors: [github.com/sponsors/pyodide](https://github.com/sponsors/pyodide)

Author: As mentioned in https://github.com/pyodide/pyodide/issues/1916#issuecomment-966209388 I made the request for validating the account yesterday, and it takes a few days to validate it. If we don't hear from them soon, we can also remove this.
